examples/barriers/inline.test.ts

```
@@ -0,0 +1,259 @@
+  // Run two credits at the same time (and wait for the deadlock)
+  const result = await Promise.race([
+    Promise.all([credit(1, 50), credit(1, 50)]).then(
+      () => "completed" as const,
+    ),
+    new Promise<"deadlock">((resolve) =>
+      setTimeout(() => resolve("deadlock"), 1000),
+    ),
+  ]);
+
+  // Terminate the stuck connections so subsequent tests can DROP TABLE
+  clients.forEach((c) => c.release(true));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)